style-about.css

```diff
@@ -0,0 +1,680 @@
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(255, 255, 255, 0.9);
+  z-index: -1;
+}
+
+.hero .a2 {
+  position: absolute;
+  bottom: 1px;
+  left: 5px;
+  width: 30px;
+}
+
+.hero .a3 {
+  position: absolute;
+  top: -29px;
+  right: -59px;
+  width: 200px;
+  z-index: 0;
+}
+
+.text-main {
+  padding-top: 22vw;
+  text-align: center;
+  color: var(--main-color-orange);
+  z-index: 2;
+}
+
+.text-main p {
+  font-size: 5vw;
+  padding-bottom: 4px;
+  font-weight: 400;
+}
+
+.text-main h1 {
+  font-family: 'cocogoose', sans-serif;
+  font-size: 7vw;
+  font-weight: 700;
+  z-index: 5;
+}
+
+.para {
+  border: 1px solid rgb(224, 221, 221);
+  background-color: white;
+  padding: 3vw;
+  width: 88vw;
+  text-align: center;
+  margin: 14px 0 13px 0;
+  color: var(--main-color-black);
+}
+
+.para p {
+  font-size: 3vw;
+  line-height: 1.6;
+  font-weight: 700;
+}
+
+.nextdate {
+  padding-bottom: 60px;
+  text-align: center;
+  margin: 5px 15px 0 30px;
+  padding-left: 20px;
+}
+
+.nextdate p {
+  font-size: 2.9vw;
+  padding: 5px 0;
+  font-weight: 400;
+  line-height: 1.6;
+  color: var(--main-color-black);
+}
+
+/* HERO MOBILE SECTION */
+
+/* FEATURED MENTORS */
+.mentors {
+  background-color: white;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  text-align: center;
+}
+
+.mentors h2 {
+  color: var(--main-color-black);
+  font-size: 5vw;
+  padding-top: 6vw;
+  padding-bottom: 5px;
+  margin: 0 20vw;
+}
+
+.mentors .line {
+  border-top: 1px solid var(--main-color-orange);
+  width: 8vw;
+  margin-top: 3vw;
+  margin-bottom: 3vw;
+}
+
+.mentors p {
+  font-size: 2.9vw;
+  padding: 5px 70px;
+  font-weight: 400;
+  line-height: 1.6;
+  color: var(--main-color-black);
+}
+
+.loremtxt {
+  display: none;
+}
+
+.mentors img {
+  border: 1px solid rgb(237, 235, 235);
+  width: 90%;
+  margin-top: 5vw;
+  margin-bottom: 6vw;
+}
+
+/* FEATURED MENTORS */
+
+/* PAST CAMPS */
+.past-camps {
+  background-color: white;
+  border-top: 1px solid rgb(237, 235, 235);
+}
+
+.past-camps p {
+  padding-bottom: 6vw;
+}
+
+.camps {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 15px;
+  padding: 10px;
+  margin-bottom: 20px;
+}
+
+.camp-card1,
+.camp-card2 {
+  border: 1px solid rgb(199, 196, 196);
+  display: flex;
+  width: 78vw;
+  height: 42vw;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background-color: rgba(184, 2, 2, 0.5);
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.camp-card1 {
+  background-image: url(./assets/cards/about_2023.png);
+}
+
+.camp-card2 {
+  background-image: url(./assets/cards/about_2021.png);
+}
+
+/* PAST CAMPS */
+
+/* OUR PARTNERS */
+.partner {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding-bottom: 4vw;
+}
```

Comment:
- [ ] As requested by the previous reviewers, make to remove the `partners` section in the `About` page in order to meet the provided Behance design.

    - ### YOUR DESIGN
    
     ![image](https://user-images.githubusercontent.com/87197412/235313667-8d3b3906-3d83-4407-99b2-ee7f7cee6b9b.png)

    - ### BEHANCE DESIGN
    
    ![image](https://user-images.githubusercontent.com/87197412/235313773-58e1f965-3a7c-4048-b260-04274b5854dc.png)